pipelines/algolia_hn_search/__init__.py

```diff
@@ -0,0 +1,127 @@
+"""
+	Loads all the search hits for specific keywords from the Algolia Hacker News Search API
+	https://hn.algolia.com/api
+"""
+import dlt
+import time
+
+from datetime import datetime
+from dlt.sources.helpers import requests
+
+
+url = "https://hn.algolia.com/api/v1/search_by_date"
+
+@dlt.source
+def algolia_hn_search(
+	keywords, # list of keywords for which the data needs to be loaded
+	start_date, # start date in datetime or "yyyy-mm-dd" format
+	end_date=datetime.today(), # end date in datetime or "yyyy-mm-dd" format
```

Comment:
you should set this default value in the code. so accept None and then assign today if None below